OpenRA.Mods.Common/Widgets/LabelWidget.cs

```diff
@@ -28,15 +29,40 @@ public class LabelWidget : Widget
 		public bool Contrast = ChromeMetrics.Get<bool>("TextContrast");
 		public Color ContrastColor = ChromeMetrics.Get<Color>("TextContrastColor");
 		public bool WordWrap = false;
+		public int LineSpacing = 2;
+
+		[Desc("line spacing = LineSpacing * font size / LineSpacingDivisor or LineSpacing if this is 0.")]
```

Comment:
I could replace these two with:
```c#
[Desc("Space between lines as a portion of line height or fixed pixel amount (rounded down).")]
public float LineSpacing = 0.2; // or 1.2 if preferred

[Desc("Whether line spacing is 'Proportional' to line height or 'Fixed' pixel amount.")]
public LineSpacingType LineSpacingType = LineSpacingType.Proportional;
```
The purpose is to keep an option for fixed pixel spacing